src/index.js

```diff
@@ -87,13 +88,19 @@ class JanusAdapter {
 
   connect() {
     debug(`connecting to ${this.serverUrl}`);
-    this.ws = new WebSocket(this.serverUrl, "janus-protocol");
-    this.session = new mj.JanusSession(this.ws.send.bind(this.ws));
-    this.ws.addEventListener("open", this.onWebsocketOpen);
-    this.ws.addEventListener("message", this.onWebsocketMessage);
+    this.ws = new WebSocketClient(this.serverUrl, "janus-protocol", {
+      strategy: 'fibonacci',
```

Comment:
I still don't think this is quite right -- the code [here](https://github.com/gamestdio/websocket/blob/master/src/index.js#L16) is looking for the strategy parameter to be named `backoff`, not `strategy`.

Incidentally, I have no particular opinion about what the backoff strategy should be. I would be interested if @robertlong or @InfiniteLee thought they had an opinion about it.